packages/medusa-core-utils/src/compare-objects.ts

```diff
@@ -0,0 +1,21 @@
+import { differenceWith, isEqual } from "lodash"
```

Comment:
thought(this-file, non-blocking): is this used anywhere in the core? Can see it has some legacy usage of `_id` from the mongo times so suspect it isn't of much use any longer.

Reply:
It seams to not be used, I removed it 👍